commands/client-kill.md

```diff
@@ -1,25 +1,23 @@
-The `CLIENT KILL` command closes a given client connection. Up to Redis 2.8.11 it was possible to close a connection only by client address, using the following form:
+The `CLIENT KILL` command closes a given client connection. This command support two formats, the old format:
 
     CLIENT KILL addr:port
 
 The `ip:port` should match a line returned by the `CLIENT LIST` command (`addr` field).
 
-However starting with Redis 2.8.12 or greater, the command accepts the following
-form:
+The new format:
 
     CLIENT KILL <filter> <value> ... ... <filter> <value>
 
 With the new form it is possible to kill clients by different attributes
 instead of killing just by address. The following filters are available:
 
 * `CLIENT KILL ADDR ip:port`. This is exactly the same as the old three-arguments behavior.
-* `CLIENT KILL ID client-id`. Allows to kill a client by its unique `ID` field, which was introduced in the `CLIENT LIST` command starting from Redis 2.8.12.
-* `CLIENT KILL TYPE type`, where *type* is one of `normal`, `master`, `slave` and `pubsub` (the `master` type is available from v3.2). This closes the connections of **all the clients** in the specified class. Note that clients blocked into the `MONITOR` command are considered to belong to the `normal` class.
+* `CLIENT KILL LADDR ip:port`. Kill all clients connected to specified local (bind) address.
+* `CLIENT KILL ID client-id`. Allows to kill a client by its unique `ID` field.
```

Comment:
i see the old text was mentioning CLIENT LIST, maybe we wanna keep that? or refer to cLIENT ID too?

Reply:
Fixed.